test/data/invalid-blank.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
What do you think about moving these fixtures to https://github.com/mapbox/mapnik-test-data? We're already using that repository for a number of fixtures here. Maybe move some of the more "problematic" ones from https://github.com/mapnik/test-data/tree/master/csv as well?